store/src/java/com/zimbra/cs/redolog/op/ModifyVolume.java

```diff
@@ -41,6 +44,8 @@ public final class ModifyVolume extends RedoableOp {
     private boolean compressBlobs;
     private long compressionThreshold;
 
+    private short storeType;
```

Comment:
I think in modify request we are not allowing users to modify its storeType, if we do so we need to fix that.

Reply:
We are calling modifyServerProperties() function line "117". This function will take care for modifying only name when storeType is 2. 
This seems okay.